data/Page/backstage.md

```diff
@@ -16,8 +14,8 @@ There are three systems you'll have access to throughout the course of the event
 
 You'll gain access to both Discord and Venueless through your ticket! The order confirmation screen has all the links you need to get in.
 
-* **If you haven't redeemed your ticket yet**, look for an email with the subject *[PyConline AU 2020] Your speaker ticket for PyConline AU 2020*, sent around TODO ADD DATE>. Then, follow the prompts to redeem your ticket.
-* **If you have redeemed your ticket already**, look for an email with the subject *[PyConline AU 2020] Your order*, sent around the time you redeemed it.
+* **If you haven't redeemed your ticket yet**, look for an email with the subject *[PyConline AU 2021] Your speaker ticket for PyConline AU 2021*, sent around TODO DATE. Then, follow the prompts to redeem your ticket.
```

Comment:
Let's stick a number in here so we have something in the calendar - how does 14 July sound? The nominal date for getting all speaker ACKs back is the 10th.